packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol

```diff
@@ -336,12 +391,23 @@ contract L1StandardBridge_DepositETH_TestFail is Bridge_Initializer {
         vm.prank(alice);
         l1StandardBridge.depositETH{ value: 1 }(300, hex"");
     }
+
+    /// @dev Tests that depositing reverts when value is passed with custom gas token.
+    function test_depositETH_customGasToken_withValue_reverts() external {
```

Comment:
We can make this into a fuzz test, it will then cover the send 0 callvalue as well as arbitrary callvalue cases